Source/UIKit/UILabel.swift

```diff
@@ -14,4 +14,8 @@ extension UILabel {
     public var rex_text: MutableProperty<String> {
         return rex_stringProperty("text")
     }
+    
+    public var rex_textColor: MutableProperty<UIColor> {
+        return rex_valueProperty(&textColor, { [weak self] in (self?.textColor)! }, { [weak self] in self?.textColor = $0 })
```

Comment:
I'm concerned about the forced cast here given that `self` is captured weakly. We should either declare this as `UIColor?` or create a static `UILabel` that we can grab the default text color from.